src/Components/Presentational/Chrome/Navigation.js

```diff
@@ -5,7 +5,7 @@ import PageWrapper from '../../Scenes/Chrome/PageWrapper/PageWrapper-Container';
 import color from '@edma/design-tokens/js/color';
 import UserGroup from '../../Scenes/SideNavigation/UserGroupListItem/UserGroupListItem';
 import DashboardListItem from '../../Scenes/SideNavigation/DashboardListItem/DashboardListItem';
-import AccessListItem from '../../Scenes/SideNavigation/AccessListItem/AccessListItem';
+import FindDataListItem from '../../Scenes/SideNavigation/FindDataListItem/FindDataListItem-Container';
```

Comment:
nitpick: `FindDataListItem` is a bit vague in my opinion. If I understand correctly, this is the search functionality within the navbar to find assets to request?

Reply:
I think your right. Let's chat about this one. I am actually going to remove the search icon from the app bar so going forward this find data in the nav will be the way to search